airflow/providers/common/sql/CHANGELOG.rst

```diff
@@ -24,6 +24,26 @@
 Changelog
 ---------
 
+1.2.0
+.....
+
+
+Features
+~~~~~~~~
+
+* ``Make placeholder style configurable (#25939)``
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Fix (and test) SQLTableCheckOperator on postgresql (#25821)``
+* ``Don't use Pandas for SQLTableCheckOperator (#25822)``
+
+.. Below changes are excluded from the changelog. Move them to
+   appropriate section above if needed. Do not delete the lines(!):
+   * ``Better error messsage for pre-common-sql providers (#26051)``
```

Comment:
I'd move those two also to the Bug-fixes section (maybe the first one to Misc section, but the second is a bug-fix for sure.